packages/react-error-overlay/package.json

```diff
@@ -38,15 +38,16 @@
     "@babel/code-frame": "^7.16.0",
     "@babel/core": "^7.16.0",
     "anser": "^2.1.0",
-    "babel-jest": "^27.4.2",
+    "babel-jest": "^28.1.3",
     "babel-loader": "^8.2.3",
     "babel-preset-react-app": "^10.0.1",
     "chalk": "^4.1.2",
     "chokidar": "^3.5.2",
     "cross-env": "^7.0.3",
     "flow-bin": "^0.116.0",
     "html-entities": "^2.3.2",
-    "jest": "^27.4.3",
+    "jest": "^28.1.3",
+    "jest-environment-jsdom": "^28.1.3",
```

Comment:
Needs to installed separately from `jest` 
https://jestjs.io/fr/docs/upgrading-to-jest28#jsdom